tests/env.rs

```diff
@@ -71,3 +71,31 @@ fn test_empty_value_is_ignored() {
 
     env::remove_var("C_A_B");
 }
+
+#[test]
+fn test_parse_numbers() {
+    env::set_var("INT_VAL", "42");
+    env::set_var("FLOAT_VAL", "42.2");
+    env::set_var("BOOL_VAL", "true");
+
+    let environment = Environment::new().try_parsing(true);
+
+    let values = environment.collect().unwrap();
+
+    assert_eq!(
```

Comment:
Maybe a beginner question, as I did not do much dev in this crate, but what is added value of this PR if you still have to use `into_int().ok()`, so parse it again to retrieve final value?

Reply:
Yeah, on the surface it looks like that for sure. Only reason I had to do this is that tests in the crate don't expose the int directly, at least as far as I understand it. Seems to be because the `Value` type exposed doesn't allow access to it's ValueKind. It's possible I could make a test that does deserialization with struct using serde instead?

Not that it's any consolation, but I have used this patch for projects of my own with success.

Reply:
I think I need to dig deeper in the code to fully grasp the added value then ;) Hope I'll find time to do it, this crate seems really nice and it would be good to ressurect it.